### PR TITLE
Document annotation container.apparmor.security.beta.kubernetes.io/*

### LINKED ISSUE #42785: Annotation `container.apparmor.security.beta.kubernetes.io` not documented / registered
**This is a Bug Report**

**Problem:**
Kubernetes uses an annotation, `container.apparmor.security.beta.kubernetes.io`, but this is not registered or documented.
We do already document an [annotation](https://kubernetes.io/docs/reference/labels-annotations-taints/#seccomp-security-alpha-kubernetes-io-pod) for seccomp.

**Proposed Solution:**
Document the annotation in [Well-Known Labels, Annotations and Taints](https://kubernetes.io/docs/reference/labels-annotations-taints/).

**Page to Update:**
https://kubernetes.io/docs/reference/labels-annotations-taints/

**Additional Information:**
/sig node security
/language en

Also see https://github.com/kubernetes/website/issues/39601

### PR DESCRIPTION
**Pull request by satyampsoni**
Fixes #42785

### CODE REVIEW

File: content/en/docs/reference/labels-annotations-taints/_index.md [hunk 1]
```diff
@@ -299,6 +299,19 @@ This annotation is part of the Kubernetes Resource Model (KRM) Functions Specifi
 which is used by Kustomize and similar third-party tools.
 For example, Kustomize removes objects with this annotation from its final build output.
 
+### container.apparmor.security.beta.kubernetes.io
```
Comment (sftim) [inline content/en/docs/reference/labels-annotations-taints/_index.md:302]:
```suggestion
### container.apparmor.security.beta.kubernetes.io/* (beta) {#container-apparmor-security-beta-kubernetes-io}
```

File: content/en/docs/reference/labels-annotations-taints/_index.md [hunk 2]
```diff
@@ -299,6 +299,19 @@ This annotation is part of the Kubernetes Resource Model (KRM) Functions Specifi
...
+Type: Annotation
+
+Example: `container.apparmor.security.beta.kubernetes.io/my-container: my-custom-profile`
+
+Used on: pods
```
Comment (sftim) [inline content/en/docs/reference/labels-annotations-taints/_index.md:307]:
```suggestion
Used on: Pods
```

File: content/en/docs/reference/labels-annotations-taints/_index.md [hunk 3]
```diff
@@ -299,6 +299,19 @@ This annotation is part of the Kubernetes Resource Model (KRM) Functions Specifi
...
+
+The `container.apparmor.security.beta.kubernetes.io/my-container annotation` allows you to specify the AppArmor security profile for a container within a Kubernetes pod. AppArmor is a Linux kernel security module that restricts the capabilities and access rights of processes within a container, enhancing the security of your workloads.
```
Comment (sftim) [inline content/en/docs/reference/labels-annotations-taints/_index.md:309]:
```suggestion
This annotation allows you to specify the AppArmor security profile for a container within a
Kubernetes pod. AppArmor is a Linux kernel security module that restricts the capabilities
and access rights of processes within a container, enhancing the security of your workloads.
```

File: content/en/docs/reference/labels-annotations-taints/_index.md [hunk 4]
```diff
@@ -299,6 +299,19 @@ This annotation is part of the Kubernetes Resource Model (KRM) Functions Specifi
...
+
+By setting this annotation, you can define a custom AppArmor profile for a container, which dictates the set of rules and restrictions that the containerized process must adhere to. This helps enforce security policies and isolation for your containers.
```
Comment (sftim) [inline content/en/docs/reference/labels-annotations-taints/_index.md:311]: Consider linking to https://kubernetes.io/docs/tutorials/security/apparmor/

**Review by sftim — COMMENTED** [review]
Thanks

Here's some feedback.

**Comment by satyampsoni** [comment]
@sftim pushed the changes!
Let me know if everything is alright.

**Comment by sftim** [comment]
/retitle Document annotation container.apparmor.security.beta.kubernetes.io/*

**Comment by sftim** [comment]
We'll make time to check this (possibly triggering a new Netlify preview build)

**Review by Arhell — COMMENTED** [review]
Please re-puch commit to restart netlify

**Pull request closed without merging**